source/common/runtime/runtime_features.cc

```diff
@@ -1,10 +1,14 @@
 #include "common/runtime/runtime_features.h"
 
+#include "common/quic/platform/flags_impl.h"
+
 #include "absl/strings/match.h"
 
 namespace Envoy {
 namespace Runtime {
 
+std::string ConstantQuicFlagPrefix = "envoy.reloadable_features.quic";
```

Comment:
I guess we don't need prefixes for SPDY or HTTP/2?

Reply:
In Envoy, spdy and http/2 flags all share the same prefix FLAGS_QUIC_RELOADABLE_FLAG_.